packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -121,6 +121,8 @@ class ContextMenu extends MenuBase {
       onLeftLastItem: null,
       onCloseRootSubmenu: null,
       onExpandLastSubmenu: null,
+      hideOnParentScroll: true,
+      visualContainer: window,
```

Comment:
Please add a `_` prefix to point out that it's private properties
And also declare `overlayContainer` and `boundaryOffset` here as well just for consistency